src/oldconfig.rs

```diff
@@ -109,6 +109,7 @@ impl OldConfigFile {
             ifdown: self.ifdown,
             ifup: self.ifup,
             ip: None,
+            advertise_addresses: None,
```

Comment:
Please remove this. Old config is only for migration from 1.x configs.

Reply:
This has to be there or the code won't compile...